src/ng-workspace.ts

```diff
@@ -15,6 +15,7 @@ export interface ProjectInfo {
   repositoryUrl: string;
   publish: boolean;
   commitPrefix: string;
+  prePublishToGit: string;
```

Comment:
I think the prePublishToGit key should be at the project level as the scripts could differ per library.